frontend/app/src/App.tsx

```diff
@@ -725,6 +729,19 @@ export class App extends PureComponent<Props, State> {
     })
   }
 
+  handleAutoRerun = (autoRerun: AutoRerun): void => {
```

Comment:
Auto rerun is implemented in a way that every frontend keeps track of the interval (inspired by Ken's auto-refresh extension), and it's an always-pull approach. Putting this logic on the backend would be possible, but potentially adds a lot more compute/memory overhead and would require a push-like execution flow... which might make things more complicated.